pkg/client/packet/client.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Is this client really needed? What do you plan to do with it?

Reply:
No, it isn't, not at all. Just a copy-paste artifact.

Reply:
I will leave the Apache license though.